agr_literature_service/lit_processing/oneoff_scripts/copy_sgd_pdfs_to_alliance.py

```diff
@@ -0,0 +1,135 @@
+import logging
+import gzip
+import shutil
+import boto3
+from botocore.exceptions import ClientError
+from os import environ, path, remove
+from agr_literature_service.lit_processing.data_ingest.dqm_ingest.utils.md5sum_utils import \
+    get_md5sum
+from dotenv import load_dotenv
+
+load_dotenv()
+
+logging.basicConfig(format='%(message)s')
+logger = logging.getLogger()
+logger.setLevel(logging.INFO)
+
+pdf_urls_file = "data/pdf_file_urls.txt"
```

Comment:
Where do these files come from ?  I don't see them in the commit

Reply:
Oh, @sweng66 says they're not meant to be in the repo, they're manual files from SGD